src/pages/Sample/SampleDebugContext.js

```diff
@@ -48,33 +47,31 @@ export async function loadData(accessionCode) {
     computedFiles,
     processorJobs,
     downloaderJobs,
-    runningJobs,
   ] = await Promise.all([
     getOriginalFiles(sample.id),
     getComputedFiles(sample.id),
     getProcessorJobs(accessionCode),
     getDownloaderJobs(accessionCode),
-    getRunningJobs(accessionCode),
   ]);
 
   return {
     sample,
     originalFiles: originalFiles.results,
     computedFiles: computedFiles.results,
-    jobs: mergeJobs(processorJobs.results, downloaderJobs.results, runningJobs),
+    jobs: mergeJobs(processorJobs.results, downloaderJobs.results),
   };
 }
 
-export function mergeJobs(processorJobs, downloaderJobs, runningJobs) {
+export function mergeJobs(processorJobs, downloaderJobs) {
   const processorJobsResult = processorJobs.map(job => ({
     ...job,
     type: 'processor',
-    isRunning: runningJobs.includes(job.nomad_job_id),
+    isRunning: job.is_queued,
   }));
   const downloaderJobsResult = downloaderJobs.map(job => ({
     ...job,
     type: 'downloader',
-    isRunning: runningJobs.includes(job.nomad_job_id),
+    isRunning: job.is_queued,
```

Comment:
It seems like you could just use `is_queued` in place of `isRunning` and not set `isRunning` at all but it's NBD.

Reply:
yeah im down for that